src/MCPClient/lib/clientScripts/archivematicaCreateMETS2.py

```diff
@@ -807,6 +766,13 @@ def createFileSec(directoryPath, parentDiv, baseDirectoryPath, baseDirectoryName
                     else:
                         dspaceMetsDMDID = ids
 
+            # If it's a .tab file, check if there's a Dataverse METS with additional metadata
+            if f.originallocation.endswith('.tab'):
+                dv_metadata = create_dataverse_tabfile_dmdsec(baseDirectoryPath, os.path.basename(f.originallocation))
+                dmdSecs.extend(dv_metadata)
+                ids = ' '.join([ds.get('ID') for ds in dv_metadata])
```

Comment:
Nitpick, but you don't need the brackets when the comprehension is the only argument to a method call.